### PR TITLE
chore: Migrated Cohort Manager to nhs repo

### PR DESCRIPTION
**Pull request by alex-clayton-1**
## Description

Moved the Cohort Manager over to NHS repo.

## Context

[DTOSS-2637](https://nhsd-jira.digital.nhs.uk/browse/DTOSS-2637)

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Comment by Mahifernando** [comment]
Could we move the 'CohortManager' directory into a high-level 'application' directory? This change will help maintain a consistent directory structure, making it easier to target multiple pipeline actions regardless of each application's internal structure.

**Review by Mahifernando — CHANGES_REQUESTED** [review]
Could we move the 'CohortManager' directory into a high-level 'application' directory? This change will help maintain a consistent directory structure, making it easier to target multiple pipeline actions regardless of each application's internal structure.

**Review by Mahifernando — COMMENTED** [review]
Please ensure that you have enabled SSH/GPG keys for commit verification. If commits are not signed, the PR will be blocked from merging.

File: CohortManager/.gitignore [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Text Editors
```
Comment (Mahifernando) [inline CohortManager/.gitignore:1]: Check if the ignore files in the apps directory are a subset of those in the root directory. If there are overlapping ignore rules, consolidate them into the root-level ignore file while keeping a necessary subset in the apps directory.

**Review by Mahifernando — APPROVED** [review]
Approve for merge 👍 

There are minor changes I can suggest, but the priority is to integrate the code with the main branch so other engineers can start working on it. Any necessary changes or refactoring should be addressed by raising a JIRA ticket in the service planning board.

However, I recommend having a second pair of eyes from the Engineering team review this PR before merging it into the main branch.

**Review by andrewmorris43 — APPROVED** [review]

**Pull request merged**